### ISSUE TITLE
Inline form errors should indicate they're errors to screen readers

### ISSUE THREAD

**Issue by anthony-kopczyk** [CMSgov/design-system#2462]
**Describe the bug**

Our team is in the process of removing 'Error:' from our inline form errors to match the form designs for medicare.gov. In the design system, error fields include an icon to indicate an error. However, this icon is not visible to screen readers, so screen readers don't have the context that the field is an error. We're concerned this is an accessibility issue from previous 508 testing on our apps.

**To Reproduce**
1. Render a TextField in the error state
2. Check the error output with a screen reader

**Expected behavior**

I expect the screen reader to indicate the message is an error of some kind.

**Screenshots**

N/A

**Additional context**

We recently brought this up on a call, so this may have already circulated a bit. Our proposed solution would be to either add alt text/title to the icon saying 'Error', or add hidden text, visible to only screen readers that says 'Error: '

**Comment by pwolfert** [comment]
Good call. We do this for other components like `Alert`, so it would make sense to provide more screen-reader context here as well. Do you already have the approved Spanish version of that prefix that you can share?

**Comment by SparrowSpit** [comment]
I'd suggest we do as Anthony mentioned above, and is [what the VA DS does](https://design.va.gov/components/form/#error-handling), and add sr-only text for "Error."

**Comment by pwolfert** [comment]
We dropped the ball on this one and forgot to add it. I've just merged the change, and it will be included in the major release going out tomorrow.
